tests/functional/test_builder.py

```diff
@@ -19,6 +19,7 @@ def setUp(self):
 
         self.source_dir = tempfile.mkdtemp()
         self.artifacts_dir = tempfile.mkdtemp()
+        self.scratch_dir = "/tmp/scratch"
```

Comment:
windows?

You should really use a `tempfile` here instead of /tmp/scratch because the test will be creating this directory. 

You can use something like `os.path.join(tempfile.mkdtemp(), "scratch")` to save yourself the headache of finding a temp filename yourself

Reply:
I didn't think of this. I first thought of doing `tempfile.mkdtemp()` but that creates the directory, which is why I did what I did. Will update.